### ISSUE TITLE
Song Recognition

### ISSUE THREAD

**Issue by Li-cyber-dot** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#259]
**Is your feature request related to a problem? Please describe.**
When I have a song stuck in my head I cannot use this Assistant to recognise the song, as shown in the screenshot, unlike the official Assistant.

**Describe the solution you'd like**
 It would be really great if some sort of music recognition algorithm was implemented just like in the official Google Assistant- although it is much easier said than done! This would I think really improve the Assistant's utility so that it can get rid of everyone's earworms!

**Additional context**

![Google_Assistant_aNwkKh42iS](https://user-images.githubusercontent.com/81527477/112881796-06fa7880-90c4-11eb-868d-bc18e592f7d2.png)

**Comment by Melvin-Abraham** [comment]
@Li-cyber-dot Seems like the feature you are referring to is Google's "Hum to Search". This feature is proprietary and does not have a public API to be consumed. Thus, this cannot be implemented as of now. Remember, this is an unofficial client and has limitations when compared to the official client.

**Comment by dangerous-sly** [comment]
Hi, 
good job for your assistant !
Like Li cyber dot, I only install it to "shazam" titles on radio or music podcast ;-( , too bad !

I keep it, it's dope anyway ! ;) and check sometimes later if the API become public !!

really good job !

**Comment by DjangoZorro** [comment]
Yeah exactly @dangerous-sly I think we got both misguided by articles like this https://rehack.com/iot/apps/nine-music-recognition-alternatives-to-shazam-for-pc/#:~:text=9.%20Google%20Assistant%20Unofficial%20Desktop%20Client

I guess it 's time to uninstall after wasting this much time to setting it up. Don't get me wrong it's not the fault of the developers btw, just the articles.

**Comment by im-coder-lg** [comment]
I think I have to tell you here. The app is _unofficial_, which means you don't get every single Google Assistant feature here. Song recognition needs some more open-source APIs and it would increase the app file till the entire app is recreated in React, which will have to be out by 2.0.0 so till then, no song recog. But if you find any FOSS song recog software for JavaScript, tell here and we'll try integrating it.

**Comment by dangerous-sly** [comment]
> Yeah exactly @dangerous-sly I think we got both misguided by articles like this https://rehack.com/iot/apps/nine-music-recognition-alternatives-to-shazam-for-pc/#:~:text=9.%20Google%20Assistant%20Unofficial%20Desktop%20Client
> 
> I guess it 's time to uninstall after wasting this much time to setting it up. Don't get me wrong it's not the fault of the developers btw, just the articles.

Yeah, I think I watch some article like this one.
I'll wait for the V 2.0.0 !
